Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -572,6 +572,7 @@ class SendAnonymousUsageDataSetting: BoolSetting {
             settingDidChange: {
                 AdjustIntegration.setEnabled($0)
                 LeanplumIntegration.sharedInstance.setEnabled($0)
+                LeanplumIntegration.sharedInstance.setUserAttributes(attributes: ["Telemetry Opt Out" : $0])
```

Comment:
For the setEnabled() function, it doesn't seem like disabling it by passing `false` will do anything (it will just bypass the check). This is related to how we can't shut it off during the app right? I can put the user attribute code before that line, but I guess this is a separate question aka I don't think it matters where I put that line of code because setEnabled won't stop the Leanplum server if we pass false; I'm assuming ideally that this function would shut Leanplum off as it's supposed to? Otherwise, it doesn't matter where the line of code goes